.github/workflows/build-docs.yml

```diff
@@ -23,10 +28,46 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
-      - name: Build docs
+      - name: Cache benchmark node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules-bm
+        with:
+          path: amagaki-benchmark/node_modules
+          key: ${{ runner.os }}-build-bm-${{ env.cache-name }}-${{ hashFiles('amagaki-benchmark/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-bm-${{ env.cache-name }}-
+      - name: Build docs and benchmark build
         run: |
                 npm install --dev
+                npm run dev:start
+
+                # Build the docs into docs/
                 npm run docs
+
+                # Build the benchmark repository.
+                cd amagaki-benchmark
+                npm install
+                node ../dist/src/index.js build
+      - uses: suisei-cn/actions-download-file@v1
```

Comment:
What does this action do... does it just `curl` the file? Maybe instead of using a separate runner you can just curl those two files directly?